perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py

```diff
@@ -181,6 +184,20 @@ def _JsonStringToPercentileResults(results, json_input, metric_name,
         metadata))
 
 
+def _GetAzureClientLibVersion(vm):
+  """ This function returns the version of azure client installed on a vm
+
+  Args:
+    vm: the VM to get the azure client version from.
+
+  Returns:
+    version: The version string of the azure python client.
```

Comment:
No need to specify the name of return variable.
Returns:
   The version string ...